--- a/zpp_bits.h
+++ b/zpp_bits.h
@@ -3292,7 +3292,7 @@ struct known_dynamic_id_variant
     ZPP_BITS_INLINE constexpr static auto serialize(auto & serializer,
                                                     auto & self)
     {
-        return serializer.template serialize_one(self.variant, self.id);
+        return serializer.serialize_one(self.variant, self.id);
     }

     Variant & variant;
@@ -3440,7 +3440,7 @@ struct [[nodiscard]] value_or_errc

     constexpr value_or_errc(value_or_errc && other) noexcept
     {
-        if (other.is_value()) {
+        if (other.success()) {
             if constexpr (!std::is_void_v<Type>) {
                 if constexpr (!std::is_reference_v<Type>) {
                     ::new (std::addressof(m_return_value))
